docs/DeveloperGuide.adoc

```diff
@@ -279,87 +279,362 @@ Refer to the guide <<DevOps#, here>>.
+
+|`* * *` |user |tag a location to each activity in the itinerary |view where each activity is taking place
+
+|`* * *` |user |edit the entries in my itinerary list |can amend any mistakes when entering data, or change my travel plans
+
+|`* * *` |user | see a chronological timeline of the activities in the itinerary | can have a clear idea of my travel plans and schedule dur
+
+|`* * *` |frequent traveler |see a record of all my past itineraries |can review my past travels
+
+|`* * *` |traveler always meeting new people overseas |add a contact that I met during my trip with metadata including their name, age, phone number, location where I met them, and their country of origin |can keep in contact with them
+
+|`* * *` |user |delete contacts |can get rid of contacts that I have not been in touch with for some time
 
-|`* * *` |user |add a new person |
+|`* * *` |user |view my itineraries, contact lists and journals specific to the trip they are relevant to |can effectively plan for multiple trips simultaneously
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |user |customize trip names |can distinguish between different trips that happen in the same location within similar date ranges
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* *` |frequent traveler |to be able to see a timeline view of all my short journal entries and photos, including the location and time of the photos and entries, per trip| ??
 
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
+|`* *` |food lover |add a meal with description, photo, price and location |keep track of what I ate
+
+|`* *` |user |sort my travel entries by category depending on the ‘feeling’ tag of each travel entry |see my best moments
+
+|`* *` |busy user |list and record items in a packing list |easily reference what I need for a trip at a glance
+
+|`* *` |fickle-minded user |delete packing list entries |remove items I feel I no longer need for the trip from the packing list
 
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
 |=======================================================================
 
-_{More to be added}_
+_{All user stories can be viewed in our issue tracker.}_
 
 [appendix]
 == Use Cases
 
-(For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
+(For all use cases below, the *System* is `Volant (V)` and the *Actor* is the `user`, unless specified otherwise)
+
+[discrete]
+=== UC01 - Adding New Trip
+
+**Precondition: **Start from home page
+
+*Guarantee:* A new Trip will be created
+
+*MSS*
+
+1. User enters details of trip to be added
+2. V writes added trip into data files and displays new trip in the trip list
```

Comment:
Use Cases should not specify what goes behind the scenes of your System (specifically adding trip into data files)